### PR TITLE
Generates the rpc and rset info for exported GDScript.

### PR DESCRIPTION
**Pull request by AndreaCatania**
- Improved the send rpc log message when fail.

This work has been kindly sponsored by IMVU.

### CODE REVIEW

File: core/io/multiplayer_api.cpp [hunk 1]
```diff
@@ -863,7 +863,7 @@ void MultiplayerAPI::_send_rpc(Node *p_from, int p_to, bool p_unreliable, bool p
 		if (method_id == UINT16_MAX && p_from->get_script_instance()) {
 			method_id = p_from->get_script_instance()->get_rpc_method_id(p_name);
 		}
-		ERR_FAIL_COND_MSG(method_id == UINT16_MAX, "Unable to take the `method_id` for the function:" + p_name + ". this can happen only if this method is not marked as `remote`.");
+		ERR_FAIL_COND_MSG(method_id == UINT16_MAX, "Unable to take the `method_id` for the function: `" + p_name + "`, node path: `" + p_from->get_path() + "`. this can happen only if this method is not marked as `remote`.");
```
Comment (akien-mga) [inline core/io/multiplayer_api.cpp]:
```suggestion
		ERR_FAIL_COND_MSG(method_id == UINT16_MAX,
				vformat("Unable to take the `method_id` for the function "%s" at path: %s. This happens when the method is not marked as `remote`.", p_name, p_from->get_path()));
```

**Comment by AndreaCatania** [comment]
Fixed and squashed commits.

File: modules/gdscript/gdscript.h [hunk 2]
```diff
@@ -185,6 +185,7 @@ class GDScript : public Script {
 	virtual void update_exports();
 
 	virtual Error reload(bool p_keep_state = false);
+	void init_rpc_methods_properties();
```
Comment (akien-mga) [inline modules/gdscript/gdscript.h]: I guess this could be private and prefixed with `_`?

Comment (AndreaCatania) [reply modules/gdscript/gdscript.h]: Fixed.

**Comment by AndreaCatania** [comment]
If anything else, this is ready to be merged @akien-mga.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**